db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200721200953) do
+ActiveRecord::Schema.define(version: 20200727032352) do
```

Comment:
I would probably keep the newest version of the schema if it were me, but I'm not positive this is correct. Maybe we could get some help with this.